pages/curriculum/ty.html

```diff
@@ -16,7 +16,7 @@
     <div class="ui container">    
     <img class="ui centered medium image" src="../../assets/images/logo.jpg">
 
-        <div class="stackable ui tabular menu">
+        <div class="stackable ui inverted bluetabular menu">
```

Comment:
Missing a space between blue and tabular

Reply:
fixed it